zerver/openapi/zulip.yaml

```diff
@@ -6207,23 +6207,7 @@ paths:
             subscribed to appropriate channels or use a shared history
             search narrow with this endpoint.
 
-            **Changes**: In Zulip 9.0 (feature level 250), narrows gained support
-            for two new filters related to channel messages: `channel` and `channels`;
-            which are aliases for (and return the same results as) the `stream` and
-            `streams` filters respectively.
-
-            In Zulip 9.0 (feature level 249), narrows gained support for a new
-            filter, `has:reaction`, which returns messages with at least one
-            [emoji reaction](/help/emoji-reactions).
-
-            In Zulip 7.0 (feature level 177), narrows gained support
-            for three new filters related to direct messages: `is:dm`, `dm` and
-            `dm-including`; replacing and deprecating `is:private`, `pm-with` and
-            `group-pm-with` respectively.
-
-            In Zulip 2.1.0, added support for using user/channel IDs when
-            constructing narrows for a message's sender, its channel and/or
-            its recipient(s).
```

Comment:
This part doesn't seem to exist at /api/construct-narrow. So it should be added there.